### PR TITLE
android-interop-testing: Support get option in interop app

### PR DESCRIPTION
**Pull request by zsurocking**

### CODE REVIEW

File: android-interop-testing/app/proguard-rules.pro [hunk 1]
```diff
@@ -14,6 +14,7 @@
 -dontwarn org.mockito.**
 -dontwarn sun.reflect.**
 -dontwarn android.test.**
+-dontwarn javax.naming.**
```
Comment (carl-mastrangelo) [inline android-interop-testing/app/proguard-rules.pro]: could you keep these sorted?

File: android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java [hunk 2]
```diff
@@ -105,12 +110,17 @@ public void onCompleted() {
 
   public InteropTester(String testCase,
                        ManagedChannel channel,
-                       TestListener listener) {
+                       TestListener listener,
+                       Boolean useGet) {
```
Comment (carl-mastrangelo) [inline android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java]: s/Boolean/boolean/

File: android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java [hunk 3]
```diff
@@ -759,4 +769,18 @@ private static boolean shouldSkip() {
     }
     return false;
   }
+
+  class SafeMethodChannelInterceptor implements ClientInterceptor {
```
Comment (carl-mastrangelo) [inline android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java]: private static final

File: android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java [hunk 4]
```diff
@@ -759,4 +769,18 @@ private static boolean shouldSkip() {
...
+    @Override
+    public <ReqT, RespT> ClientCall<ReqT, RespT> interceptCall(MethodDescriptor<ReqT, RespT> method,
```
Comment (carl-mastrangelo) [inline android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java]: group all args together on the same line.

File: android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java [hunk 5]
```diff
@@ -759,4 +769,18 @@ private static boolean shouldSkip() {
...
+        CallOptions callOptions, Channel next) {
+      return new CheckedForwardingClientCall<ReqT, RespT>(next.newCall(
+          method.toBuilder().setSafe(true).build(), callOptions)) {
```
Comment (carl-mastrangelo) [inline android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java]: similar comment here.  Line breaking happens at the highest level.  I would suggest:

```
      return new CheckedForwardingClientCall<ReqT, RespT>(
          next.newCall(method.toBuilder().setSafe(true).build(), callOptions)) {
```

File: android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java [hunk 6]
```diff
@@ -759,4 +769,18 @@ private static boolean shouldSkip() {
...
+        @Override
+        public void checkedStart(final Listener<RespT> responseListener, Metadata headers) {
```
Comment (carl-mastrangelo) [inline android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java]: optional: I would remove final.  It doesn't add any value and it clutters the code.

File: android-interop-testing/app/proguard-rules.pro (hunk 1 above)
Comment (zsurocking) [inline android-interop-testing/app/proguard-rules.pro]: done

File: android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java (hunk 2 above)
Comment (zsurocking) [inline android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java]: done

File: android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java (hunk 3 above)
Comment (zsurocking) [inline android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java]: done

File: android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java (hunk 4 above)
Comment (zsurocking) [inline android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java]: done

File: android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java (hunk 5 above)
Comment (zsurocking) [inline android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java]: done

File: android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java (hunk 6 above)
Comment (zsurocking) [inline android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java]: removed

**Pull request merged**